tests/test_drosophila_melanogaster.py

```diff
@@ -0,0 +1,58 @@
+"""
+Tests for the drosophila_melanogaster data definitions.
+"""
+import unittest
+import io
+
+import msprime
+
+from stdpopsim import drosophila_melanogaster
+
+
+class TestGenome(unittest.TestCase):
+    """
+    Tests for the drosophila_melanogaster genome.
+    """
+    def test_basic_attributes(self):
+        genome = drosophila_melanogaster.genome
+        self.assertEqual(genome.species, "drosophila_melanogaster")
+        self.assertEqual(genome.default_genetic_map, "Comeron2012_dm6")
+        self.assertEqual(len(genome.chromosomes), 8)
+
+    def test_str(self):
+        s = str(drosophila_melanogaster.genome)
+        self.assertGreater(len(s), 0)
+        self.assertIsInstance(s, str)
+
+    def test_chromosome_lengths(self):
+        genome = drosophila_melanogaster.genome
+        # Numbers from https://www.ncbi.nlm.nih.gov/grc/human/data
```

Comment:
This comment is misleading, right? Should just be "Numbers from dm6"

Reply:
yep. ugh didn't clean out the comments after I started working from the Homo_sapiens example